docs/getting-started/create-remote-module.md

```diff
@@ -1,11 +1,12 @@
 ---
 order: 90
-label: Create remote module
 ---
 
 # Create a remote module
 
-By adding a remote module, we'll transform our existing host application into a truly distributed application.
+Remote modules are modules that are not part of the current build but are **loaded at runtime** from a remote server. They are useful because they enables teams be **fully autonomous** by **deploying** their modules **independently** from the other parts of the application.
```

Comment:
```suggestion
Remote modules are modules that are not part of the current build but are **loaded at runtime** from a remote server. They are useful because they enable teams to be **fully autonomous** by **deploying** their modules **independently** from the other parts of the application.
```